### PR TITLE
App Dev Guide should not use git or repo

### PR DESCRIPTION
**Pull request by ojalatodd**
Removed use of git clone in the App Dev Guide,
Environment setup section.

Signed-off-by: Todd Ojala <todd@bitwise.io>

### CODE REVIEW

**Comment by rbuysse** [comment]
I renamed this to sawtooth-default.yaml in a PR just before yours. Can you update the references?

https://github.com/hyperledger/sawtooth-core/pull/476

**Comment by ojalatodd** [comment]
@rbuysse Windows seems to use a different build file or process, i.e., make.bat instead of Makefile. I'd like to get your help with creating a make.bat that does what the Makefile now does, by copying the docker compose yaml file to $(BUILDDIR)/html/app_developers_guide/sawtooth-demo.yaml

**Comment by ojalatodd** [comment]
This PR might conflict with #476 . Rebase and resolve conflicts before merging (note to self)

File: docs/Makefile [hunk 1]
```diff
@@ -64,6 +64,7 @@ html: templates cli
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@cp ../docker/compose/sawtooth-demo.yaml $(BUILDDIR)/html/app_developers_guide/sawtooth-demo.yaml
```
Comment (rbuysse) [inline docs/Makefile]: this should be sawtooth-default.yaml

File: docs/source/app_developers_guide/environment_setup.rst [hunk 2]
```diff
@@ -30,14 +30,21 @@ The following tools are required:
 * `Docker Compose <https://docs.docker.com/compose/install/>`_ (Linux only)
 
 
-Step One: Clone Repository
---------------------------
+Get the Docker Yaml File
+------------------------
 
+<<<<<<< fb9a98b2f7290d9e087e163c4ff450302946484f
```
Comment (rbuysse) [inline docs/source/app_developers_guide/environment_setup.rst]: Unresolved merge conflict here...

File: docs/source/app_developers_guide/environment_setup.rst [hunk 3]
```diff
@@ -67,17 +74,24 @@ Open up a terminal and run the following:
...
 
 
 Environment Startup
 -------------------
 
-To start up the environment, run:
+To start up the environment, perform the following tasks:
+
+1. Open a terminal window.
+2. Change your working directory to the same directory where you saved the
```
Comment (rbuysse) [inline docs/source/app_developers_guide/environment_setup.rst]: Might want to include and example of what you mean here. Something like "($ cd ~/Downloads)"

File: docs/Makefile (hunk 1 above)
Comment (ojalatodd) [inline docs/Makefile]: Fixed.

File: docs/source/app_developers_guide/environment_setup.rst (hunk 2 above)
Comment (ojalatodd) [inline docs/source/app_developers_guide/environment_setup.rst]: Fixed

File: docs/source/app_developers_guide/environment_setup.rst [hunk 4]
```diff
@@ -30,54 +30,31 @@ The following tools are required:
 * `Docker Compose <https://docs.docker.com/compose/install/>`_ (Linux only)
 
 
-Step One: Clone Repository
---------------------------
+Get the Docker Yaml File
```
Comment (aludvik) [inline docs/source/app_developers_guide/environment_setup.rst]: Change to "Get the Docker Compose File"

File: docs/source/app_developers_guide/environment_setup.rst [hunk 5]
```diff
@@ -30,54 +30,31 @@ The following tools are required:
...
+------------------------
 
-You'll need to have git installed in order to clone the Sawtooth source
-code repository. You can find up-to-date installation instructions here:
+A docker compose file is provided which defines the sawtooth environment,
```
Comment (aludvik) [inline docs/source/app_developers_guide/environment_setup.rst]: "...defines the *process for constructing a* sawtooth environment..."

File: docs/source/app_developers_guide/environment_setup.rst (hunk 3 above)
Comment (ojalatodd) [inline docs/source/app_developers_guide/environment_setup.rst]: Added an example

File: docs/source/app_developers_guide/environment_setup.rst (hunk 4 above)
Comment (ojalatodd) [inline docs/source/app_developers_guide/environment_setup.rst]: Changed

File: docs/source/app_developers_guide/environment_setup.rst (hunk 5 above)
Comment (ojalatodd) [inline docs/source/app_developers_guide/environment_setup.rst]: Changed

**Pull request merged**